titus-server-federation/src/main/java/com/netflix/titus/federation/service/AggregatingJobServiceGateway.java

```diff
@@ -493,10 +499,12 @@ private JobChangeNotification addStackName(JobChangeNotification notification) {
         switch (notification.getNotificationCase()) {
             case JOBUPDATE:
                 Job job = addStackName(notification.getJobUpdate().getJob());
+                job = JobEventPropagationUtil.recordChannelLatency(CHECKPOINT_FED_CLIENT, job, notification.getTimestamp(), titusRuntime.getClock());
```

Comment:
Presumably this is something we should add to FedV2 as well so that we don't lose it later in the quarter when we switch to that?

Reply:
Yes. Once we get this working and cleaned up, I would like to add it to FedV2 as well.